src/math.js

```diff
@@ -65,6 +65,46 @@ engine.mod = function(x, y){
   return x % y;
 };
 
+engine.abs = function(x){
+  if (isEmpty(x)){
+    return [];
+  }else{
+    let num = ensureNumberSingleton(x);
+    if (typeof num != 'number'){
+      return num;
```

Comment:
"When taking the absolute value of a quantity, the unit is unchanged."
(http://hl7.org/fhirpath/2019May/index.html#abs-integer-decimal-quantity)
Quantity as an array with a single string element. The only way to recognize quantity is to implement toQuantity() function (http://hl7.org/fhirpath/2019May/index.html#toquantityunit-string-quantity).
So, should I implement toQuantity() in misc.js and then import it to math.js?

Reply:
I think abs() function should look something like that:

```
engine.abs = function(x){
  if (isEmpty(x)){
    return [];
  }else{
    if(x.toQuantity()){
      return x;
    }
    let num = ensureNumberSingleton(x);
    return Math.abs(num);
  }
};
```

Reply:
Also, all of the string data would be recognized as quantity if they are convertable to quantity and I don't know how to set a quantity variable in my tests.

Reply:
> So, should I implement convertsToQuantity() in misc.js and then import it to math.js?
No, because I am working on adding Quantity (to add support for adding quantities to date/time).  So, hold off on anything involving Quantity; you might leave yourself a "TBD" comment to add code for those after my changes are in.

Reply:
I will see if I can separate my PR so that the Quanitity type is defined in one PR, and the date/time + quantity stuff (which might take me a while) is in another.